### PR TITLE
WGFAST 2024 updates

### PR DESCRIPTION
**Pull request by valentina-s**

### CODE REVIEW

File: content/newslist.dat [hunk 1]
```diff
@@ -1,3 +1,7 @@
+**[04/2024]** Wu-Jung and Valentina presented two talks at the [WGFAST 2024](https://www-iuem.univ-brest.fr/wgfast/?lang=en) meeting in France on pipelines for echosounder data processing (on ship & cloud).
```
Comment (leewujung) [inline content/newslist.dat:1]:
```suggestion
**[04/2024]** Wu-Jung and Valentina presented two talks at the [WGFAST 2024](https://www-iuem.univ-brest.fr/wgfast/?lang=en) meeting in France on pipelines and software tools for echosounder data processing on both ship and cloud.
```
We can probably create pages with abstract and maybe 1 fig and link from the news.

File: content/newslist.dat [hunk 2]
```diff
@@ -1,3 +1,7 @@
+**[04/2024]** Wu-Jung and Valentina presented two talks at the [WGFAST 2024](https://www-iuem.univ-brest.fr/wgfast/?lang=en) meeting in France on pipelines for echosounder data processing (on ship & cloud).
+
+**[04/2024]** Wu-Jung presented on [Echosounder Data Processing Levels](https://echolevels.readthedocs.io/en/latest/) (with contributions from Emilio, Brandyn, and Valentina) at the Global Acoustics INteroperable workshop associated with the WGFAST 2024 meeting.
```
Comment (leewujung) [inline content/newslist.dat:3]: Change the link to the page that actually contains the proposed levels.
```suggestion
**[04/2024]** Wu-Jung presented on [Echosounder Data Processing Levels](https://echolevels.readthedocs.io/en/latest/levels_proposed.html) (with contributions from Emilio, Brandyn, and Valentina) at the Global Acoustics INteroperable (GAIN) workshop associated with the WGFAST 2024 meeting.

```

**Review by leewujung — APPROVED** [review]
Thanks for putting these together!

File: content/newslist.dat (hunk 1 above)
Comment (valentina-s) [reply content/newslist.dat:1]: Should we just link to the slides? Seems a lot of pages to make for each abstract? We can also link to the slides from the individual project descriptions.

Comment (valentina-s) [reply content/newslist.dat:1]: Now that I am looking through the website I realize that we had a whole talks section. Ok, I will make pages, but maybe in the future we reorganize a bit so those are linked to the projects, and people can read more on the projects.

**Pull request merged**